app/views/vaccines/index.html.erb

```diff
@@ -3,13 +3,11 @@
 <% @vaccines.each do |vaccine| %>
   <%= render AppCardComponent.new do |c| %>
     <% c.with_heading { link_to(vaccine_heading(vaccine), vaccine_path(vaccine)) } %>
-    <p class="nhsuk-body-s nhsuk-secondary-text-color">
+    <p class="nhsuk-body">
```

Comment:
I don't think `nhsuk-body` does anything in our app because we have global styles for paragraphs 🤔

Reply:
Fair enough, will remove.